functions/src/v1/utils/BookingUtils/BookingUtils.ts

```diff
@@ -37,10 +37,8 @@ export function calculateCarSpaceCapacity(
   if (isBookingDateLimitedToBU(dateToBook) && !businessUnit) {
     throw Error('If booking date is limited to BU then BU is required');
   }
-  // TODO: revert after hawking join date
-  const hawkingCarCapacity = isBeforeHawkingJoin(dateToBook)
-    ? 0
-    : defaults.hawkingCarCapacity;
+
+  const adamsCarCapacity = defaults.adamsCarCapacity;
```

Comment:
Nitpick - is this variable needed?

Reply:
probably not. Will remove it

Reply:
spotted that the other values in the if statement were just used directly so thought we may as well be consistent :)